src/test/codegen/integer-cmp.rs

```diff
@@ -0,0 +1,32 @@
+// This is test for more optimal Ord implementation for integers.
+// See <https://github.com/rust-lang/rust/issues/63758> for more info.
+
+// compile-flags: -C no-prepopulate-passes -C opt-level=3
```

Comment:
You’ll want to ensure this test is compiled with optimisations. Remove `-Cno-prepopulate-passes`.

Reply:
Nice. Thank you.